lib/main.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('lodash');
-var css = require('css');
+var postcss = require('postcss');
```

Comment:
Yes, `postcss` is the same CSS parser that `autoprefixer` uses, and `css-what` is the same selector parsing engine that `cheerio` uses.